lib/beacon/config.ex

```diff
@@ -48,6 +48,14 @@ defmodule Beacon.Config do
   """
   @type template_formats :: [{format :: atom(), description :: String.t()}]
 
+  @type asset :: [{MIME.t(), mime_config()}]
```

Comment:
I don't think `MIME.t/0` exists, it's just a `String.t/0` :)
but we can have our own custom type

Reply:
You're right. I meant to ask if you wanted anything special there and that was a place keeper.

Reply:
`String.t()` or a custom `ContentType.t()` works for me, whatever you prefer 👍🏻